### ISSUE TITLE
The SDK 'Microsoft.NET.Sdk.Web' specified could not be found.

### ISSUE THREAD

**Issue by jairbubbles** [Microsoft/msbuild#2465]
I'm using NuGet package version 15.3.409 to load `.csproj` using the `Project` API. The code is very minimalistic:

```csharp
using Microsoft.Build.Evaluation;

void LoadProject(string projectPath)
{
    var msBuildProject = ProjectCollection.LoadProject(projectPath);
}
```

Since I installed Visual Studio 2017 update 3 I'm getting an exception `The SDK 'Microsoft.NET.Sdk.Web' specified could not be found.` when trying to load a project that targets **ASP.NET Core**.

I've read many posts on the subject but it seems to apply to msbuild.exe not the NuGet's library.

**Comment by rainersigwald** [comment]
Duplicate of #2369.

The symptoms look different, but I'm pretty sure this is the same underlying issue.

Please see that issue for workaround information.

**Comment by Tragetaschen** [comment]
I've been running into this today and couldn't figure out how to fix this after going through #2369:
- I've tried to reference the myget version `15.5.0-preview-000090-0999971`: same error as above
- I've started to set `VSINSTALLDIR` and `VisualStudioVersion`. Now it finds the SDK, but  searches in the wrong folder for `Microsoft.Common.props`:
```
Unhandled Exception: Microsoft.Build.Exceptions.InvalidProjectFileException: The imported project 
"C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\15.0\Bin\15.0\Microsoft.Common.props" 
was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.  
C:\Program Files\dotnet\sdk\2.0.0\Sdks\Microsoft.NET.Sdk\Sdk\Sdk.props
```
There's an additional `15.0\Bin` that's not on my disk. There's also a ` C:\Program Files\dotnet\sdk\2.0.0\15.0\Microsoft.Common.props` that seems like a better fit...

#2369 suggests that this error has something to do with Runtime package, but I've never referenced that in my entire life :)
